src/main/java/com/crud/kodilla_library/controller/RentalController.java

```diff
@@ -0,0 +1,48 @@
+package com.crud.kodilla_library.controller;
+
+import com.crud.kodilla_library.controller.exceptions.RentalNotFoundException;
+import com.crud.kodilla_library.domain.dto.RentalDto;
+import com.crud.kodilla_library.service.RentalDbService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/library/rentals")
+@RequiredArgsConstructor
+public class RentalController {
+
+    private final RentalDbService rentalDbService;
+
+    @GetMapping
+    public List<RentalDto> getRentals() {
+        return rentalDbService.getAllRentals();
+    }
+
+    @GetMapping(value = "{rentalId}")
+    public RentalDto getRental(@PathVariable long rentalId) throws RentalNotFoundException {
```

Comment:
miales zrobic RuntimeExceptiony

Reply:
Okej, myślałem że mam wybór jak to rozwiązać ;)